### ISSUE TITLE
Training Output of the Network 

### ISSUE THREAD

**Issue by ghost** [zoujiemeng/wave_cnn#1]
Hi !
I am very new to neural network. I tried to compile the code by using Code Blocks (but without the Makefile provided as Code Blocks has pre/post build option). By using the example.c code provided, after commented out the macro definitions and entered the path of the test dataset and training dataset provided, I managed to get the following training results.
 
![capture](https://user-images.githubusercontent.com/24191116/33513120-655e057c-d777-11e7-9632-86e20d656e9c.PNG)

I would like to know whether I'm on the right track by getting the results included in the screenshot above as I didn't use the Makefile provided. Any help would be much appreciated. Thank you so much!

**Comment by zoujiemeng** [comment]
there is on doubt that you are on the right way. In fact, the result were not as good as expected. It is confirmed that unbalanced training datasets led to bad result because I used Tensorflow to train the same datasets in the same network and it's accuracy was still around 73.9%. I've tried to improve my datasets , but to mark every wave segment is too time consuming... well, in short, you're on the right way.
